Makefile

```diff
@@ -172,9 +175,10 @@ src/rust/gen/analyzer0f.rs: $(ANALYZER_DEPENDENCIES)
 build/v86.wasm: $(RUST_FILES) build/softfloat.o build/zstddeclib.o Cargo.toml
 	mkdir -p build/
 	-BLOCK_SIZE=K ls -l build/v86.wasm
+	RUSTFLAGS="--emit asm"
 	cargo rustc --release $(CARGO_FLAGS)
 	cp build/wasm32-unknown-unknown/release/v86.wasm build/v86.wasm
-	-$(WASM_OPT) && wasm-opt -O2 --strip-debug build/v86.wasm -o build/v86.wasm
+	-$(WASM_OPT) && wasm-opt -O4 --strip-debug build/v86.wasm -o build/v86.wasm
```

Comment:
Also check that this doesn't increase the size of v86.wasm too much.

Reply:
it's for performance, not size, and yes my internet connection can easily handle it